packages/react-integration/demo-app-ts/src/components/demos/ExpandableSectionDemo/ExpandableSectionDemo.tsx

```diff
@@ -81,6 +81,16 @@ export class ExpandableSectionDemo extends React.Component<null, ExpandableSecti
         >
           This content is visible only when the component is expanded.
         </ExpandableSection>
+        <br />
```

Comment:
We don't really need to add this demo since there is no related test case.  We have recently changed what we are writing for demos since Zack made the effort to clean up integration test.  With the shift to TS for examples, the demo-app will eventually not be needed and we can test against our example code.